### ISSUE TITLE
segmentation fault on Linux Ubuntu 20.04

### ISSUE THREAD

**Issue by mattvenn** [trilomix/GDS3D#20]
I tried running the example demo and immediately get a segfault. 
Any info I can provide to help debug?

**Comment by ga47wot** [comment]
Yes same for me as well. I get segmentation fault in linux. The provided example runs on Windows though. But the program does not start on even windows when I am using my own porcess techfile and gds file. Were you able to solve this problem on linux (segmentation fault) or on windows ? I did try increasing the maxlayers field (>2048) inside the gdsparse.h file but it did not help me.
